x-pack/plugins/cases/server/authorization/index.ts

```diff
@@ -101,6 +102,14 @@ export const Operations: Record<ReadOperations | WriteOperations, OperationDetai
     docType: 'case',
     savedObjectType: CASE_SAVED_OBJECT,
   },
+  [WriteOperations.PushCase]: {
+    type: EVENT_TYPES.change,
+    name: WriteOperations.PushCase,
+    action: 'push-case',
+    verbs: updateVerbs,
```

Comment:
nit?: Should the push have separate verbs defined? 'push, pushing, pushed'?

Reply:
Hmm 🤔  good point, right now the verbs are tied to the valid ECS event types that we use `access, creation, deletion, change` (push isn't a valid one). Pushing a case does act as a change as it could close the case and it changes the push_at timestamp. I think for now I'm going to stick with the update verbs but I'll make a note of it and discuss it with Christos when he gets back.